README.md

```diff
@@ -50,5 +50,6 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [WeDo Technologies](https://www.wedotechnologies.com/) [:rocket:](https://www.wedotechnologies.com/en/careers/) | Revenue Assurance and Fraud Management solutions. | `Braga` `Lisboa` |
 | [WIT Software](https://www.wit-software.com) [:rocket:](https://www.wit-software.com/careers/) | Mobile telecommunications services. | `Aveiro` `Coimbra`<br>`Leiria` `Lisboa`<br>`Porto` |
 | [YLD](https://www.yld.io/) [:rocket:](https://www.yld.io/join-us/) | Software engineering, design, training and open-source. | `Lisboa` `Porto`<br>`Remote` |
+| [YouClap](https://youclap.tech) | Create, participate and share awesome challenges. | `Aveiro` `Remote` |
```

Comment:
Como assim, remote? 😛😂

Reply:
you can work from the balcony in our office 😂 or from wherever you feel more comfortable 😜

Reply:
😂😂

Reply:
:thinking:

Reply:
🙃 i'm kidding 😂